sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/ServicePrincipalImpl.java

```diff
@@ -291,9 +293,9 @@ public ServicePrincipalImpl withNewApplication(Creatable<ActiveDirectoryApplicat
     }
 
     @Override
-    public ServicePrincipalImpl withNewApplication(String signOnUrl) {
```

Comment:
Maybe better to keep the overload taking the `String`.

Reply:
Added back for it.